main.go

```diff
@@ -419,13 +419,18 @@ Generate an identity seed and launch a gateway:
 			EnvVars: []string{"RAINBOW_HTTP_RETRIEVAL_ALLOWLIST"},
 			Usage:   "When HTTP retrieval is enabled, allow it only to the given hosts. Empty means 'everyone'",
 		},
+		&cli.StringSliceFlag{
+			Name:    "http-retrieval-denylist",
+			Value:   cli.NewStringSlice(),
+			EnvVars: []string{"RAINBOW_HTTP_RETRIEVAL_DENYLIST"},
+			Usage:   "When HTTP retrieval is enabled, the given hosts are never contacted.",
```

Comment:
are these full urls including the scheme? e.g. `https://dag.w3s.link`? Or just the hostname `dag.w3s.link`?

Reply:
just hostname, I will add an example